### PR TITLE
Move text messages to JSON

### PR DESCRIPTION
**Pull request by totop716**
- Move text messages of account details component to JSON
- Fix analytics pair details page
- Fix audit issues

### CODE REVIEW

File: src/components/TransactionsTable/TransactionsTable.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
 import { TxnType } from 'constants/index';
 import 'components/styles/TransactionsTable.scss';
+dayjs.extend(relativeTime);
```
Comment (FriskyHamTitz) [inline src/components/TransactionsTable/TransactionsTable.tsx:12]: Why is this here?

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]
See comments

File: src/components/TransactionsTable/TransactionsTable.tsx (hunk 1 above)
Comment (totop716) [reply src/components/TransactionsTable/TransactionsTable.tsx:12]: This is to call `fromNow` function of `dayjs`. Currently it is returning error on analytics page because we are missing this line

File: package.json [hunk 2]
```diff
@@ -100,7 +100,8 @@
     "async": "^3.2.2",
     "ejs": "^3.1.7",
     "cross-fetch": "^3.1.5",
-    "eventsource": "^2.0.2"
+    "eventsource": "^2.0.2",
+    "protobufjs": "^6.11.3"
   },
```
Comment (FriskyHamTitz) [inline package.json:105]: Why is protobufjs added?

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]
Why was protobufjs added

File: package.json (hunk 2 above)
Comment (totop716) [reply package.json:105]: This was added to fix audit issue.

**Review by FriskyHamTitz — APPROVED** [review]
